guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,24 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
+
+.Procedure
+. Disable Puppet server on external {SmartProxies} before you disable it on {ProjectServer}:
```

Comment:
Perhaps emphasize it's on all? Though we have coded a safe guard, it doesn't hurt to be explicit.
```suggestion
. Disable Puppet server on all external {SmartProxies} before you disable it on {ProjectServer}:
```